test/plugins/directories_command_test.exs

```diff
@@ -80,16 +80,6 @@ defmodule DirectoriesCommandTest do
       {:validation_failure, {:bad_argument, "Cannot set both online and offline"}}
   end
 
-  test "validate_execution_environment: when --offline is used, not specifying an enabled_plugins_file fails validation", context do
```

Comment:
Why were these tests removed? Testing execution environment validation is too time consuming to do manually.